### ISSUE TITLE
A window in floating mode dragged from one monitor to another, auto-tiles

### ISSUE THREAD

**Issue by julianlam** [pop-os/pop#1426]
**Distribution (run `cat /etc/os-release`):**
 ```
NAME="Pop!_OS"
VERSION="20.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.04 LTS"
VERSION_ID="20.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
LOGO=distributor-logo-pop-os
``` 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
Tiling manager? Unsure of package name

**Issue/Bug Description:**
This seems to be a regression. "Tile Windows" is on. When dragging a floating window from one monitor to another, the window loses its "floating" status and attaches to one side of the screen as though it were tiled.

Interestingly, the tiling manager *still thinks that window is floating*. I must hit `Super-G` twice to have the window float again.

**Steps to reproduce (if you know):**

1. Attach an external monitor to your computer
1. Engage tiling mode
1. Float a window (in my case, it was slack)
1. Attempt to drag the floating window from one monitor to another (I used alt-click to drag, but dragging via window toolbar also reproduces)
1. The window will attach to the side of the destination monitor once it crosses the threshold into that monitor

**Expected behavior:**

The window should remain floating.

**Comment by julianlam** [comment]
This regression seems to have resolved itself as of `1.1.0~1608318806~20.04~19ebae0`

**Comment by julianlam** [comment]
Well, _resolved itself_ is not correct, I'm sure _someone_ resolved it. Thanks, whoever did that :smile:
